src/Arcus.WebApi.Tests.Runtimes.AzureFunction/Startup.cs

```diff
@@ -16,7 +15,7 @@ public class Startup : FunctionsStartup
         /// <param name="builder">The instance to build the registered services inside the functions app.</param>
         public override void Configure(IFunctionsHostBuilder builder)
         {
-            builder.AddHttpCorrelation(configureOptions: (Action<HttpCorrelationInfoOptions>) null);
+            builder.AddHttpCorrelation(options => options.Format = HttpCorrelationFormat.Hierarchical);
```

Comment:
Couldn't do any better. Had some problems with Docker <> W3C <> Azure Functions, so changed this to hierarchical for testing. The service is tested separately.

Reply:
Weird, what kind of problem ?

Reply:
I got it to work manually, but via the YAML build, I keep having a `TelemetryClient` not found, or a `.dll` not found exception. Tried several things for more than a week, so updated this eventually to hierarchical. It is tested separately, and is also in-process (which will be deprecated).

Reply:
Weird, are we sure we're not going to run into this when using it in a real project ?

Reply:
I got it to work locally without Docker, as my Docker knowledge is too limited very this. I think it has to do with the restore operation of packages and that it uses different versions when building via Docker. But some help would be helpful. Feel free to check this yourself. 👍

Reply:
> and that it uses different versions when building via Docker

That would be very strange.
I suggest that we complete this PR and look at it together; that might be most productive.  Maybe when we're both at the office ?

Reply:
My days at the office are still very limited and I can't say for sure when the next time will be. Definitely a couple weeks from now. It is the remote build that is failing, so the only errors are the logs from the build machine.

Reply:
Ok, I think I found something. It has something to do with an internal conflict of app insights packages.
in-process Azure Functions may need to use this package: https://learn.microsoft.com/en-us/azure/azure-functions/functions-dotnet-class-library?tabs=v2%2Ccmd#log-custom-telemetry-in-c-functions

Reply:
Great find.  Eager to know if that fixes the issue or not.

Reply:
It does, and the change has already be pushed here.